scout/server/blueprints/cases/controllers.py

```diff
@@ -689,7 +711,9 @@ def case_report_variants(store: MongoAdapter, case_obj: dict, institute_obj: dic
     data["variants"] = evaluated_variants_by_type
 
 
-def _get_decorated_var(var_obj: dict, institute_obj: dict, case_obj: dict) -> dict:
+def _get_decorated_var(
```

Comment:
I'm not super fond of a function named get decorated - the latter clashes with "decorator" in my mind. But enough refactoring here I think.